tools/clang/test/HLSLFileCheck/hlsl/types/struct/bitfields.hlsl

```diff
@@ -0,0 +1,139 @@
+// RUN: %dxc /Zi /T vs_6_2 /E main %s | FileCheck %s
+
+// CHECK: %struct.foo = type { i32 }
+// CHECK: %struct.S = type { i32 }
+// CHECK: %struct.P1 = type { i32, i32 }
+
+RWByteAddressBuffer BufferOut : register(u0);
+
+// The store of p.y into the temporary was not
+// getting extended to 32 bits, so uninitialized
+// bits of the temporary were used.  7366161.
+struct foo {
+  int x : 8;
+  int : 8;
```

Comment:
Wait, so anonymous bitfield *is* supported?  What's with the (unchecked) errors in the other test that read `clang: error: anonymous bit-field`?

Reply:
I suspect that dxc, unlike clang, does implicit type conversions (to the basis type) in those cases.  Would there be a legitimate reason for that?